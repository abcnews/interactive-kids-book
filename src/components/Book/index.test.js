const React = require('react');
const renderer = require('react-test-renderer');

const Book = require('.');

describe('Book', () => {
  test('It renders', () => {
    const component = renderer.create(<Book />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
