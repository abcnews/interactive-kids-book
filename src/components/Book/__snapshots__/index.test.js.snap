// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Book It renders 1`] = `
<div
  className="wrapper"
  style={
    Object {
      "height": 768,
      "width": 1024,
    }
  }
/>
`;
